pwa/app/components/tba/teamEventAppearance.tsx

```diff
@@ -53,11 +76,172 @@ export default function TeamEventAppearance({
   );
 }
 
-function TeamStatus({ status }: { status: TeamEventStatus | null }) {
-  // todo: build this out with better mid-event, future-event logic handling
+function TeamStatus({
+  status,
+  team,
+  awards,
+  maybeDistrictPoints,
+  maybeAlliances,
+}: {
+  status: TeamEventStatus | null;
+  team: Team;
+  awards: Award[];
+  maybeDistrictPoints: EventDistrictPoints | null;
+  maybeAlliances: EliminationAlliance[] | null;
+}) {
   return (
-    <div
-      dangerouslySetInnerHTML={{ __html: status?.overall_status_str ?? '' }}
-    />
+    <div className="flow-root">
+      <dl className="-my-3 divide-y divide-gray-300 text-sm">
+        {status?.qual?.ranking?.rank && (
+          <div className="grid grid-cols-1 gap-1 py-3 sm:grid-cols-3 sm:gap-4">
+            <dt className="font-medium text-gray-900">Rank</dt>
+            <dd className="text-gray-700 sm:col-span-2">
+              {status.qual.ranking.rank} / {status.qual.num_teams}
+            </dd>
+          </div>
+        )}
+
+        {status?.qual?.ranking?.record && (
+          <div className="grid grid-cols-1 gap-1 py-3 sm:grid-cols-3 sm:gap-4">
+            <dt className="font-medium text-gray-900">Record</dt>
+            <dd className="text-gray-700 sm:col-span-2">
+              {status.qual.ranking.record.wins +
+                (status.playoff?.record?.wins ?? 0)}
+              -
+              {status.qual.ranking.record.losses +
+                (status.playoff?.record?.losses ?? 0)}
+              -
+              {status.qual.ranking.record.ties +
+                (status.playoff?.record?.ties ?? 0)}
+            </dd>
+          </div>
+        )}
+
+        {awards.length > 0 && (
+          <div className="grid grid-cols-1 gap-1 py-3 sm:grid-cols-3 sm:gap-4">
+            <dt className="font-medium text-gray-900">
+              {pluralize(awards.length, 'Award', 'Awards', false)}
+            </dt>
+            <dd className="text-gray-700 sm:col-span-2">
+              <ul
+                className={cn({
+                  'list-none': awards.length == 1,
+                  'list-disc': awards.length > 1,
+                })}
+              >
+                {awards.map((award) => {
+                  const namedRecipients = award.recipient_list.filter(
+                    (r) =>
+                      r.awardee !== null &&
+                      r.awardee !== '' &&
+                      r.team_key === team.key,
+                  );
+
+                  return (
+                    <li key={`${award.award_type}-${award.event_key}`}>
+                      {award.name}
+                      {namedRecipients.length > 0 ? (
+                        <>
+                          {' '}
```

Comment:
Is this leading space necessary?

Reply:
Yes, it creates the space between the award and the names:

![image](https://github.com/user-attachments/assets/5b1ac5a9-40a4-4654-a1b0-a5cba1897c66)